### ISSUE TITLE
Fix the H1

### ISSUE THREAD

**Issue by mbakeryo** [mlibrary/heliotrope#1751]
Use proper heading structure and ensure that status of interactive elements (e.g., collapsed/expanded) are identified correctly to allow for various/familiar ways to navigate and interact with a page. 

On publisher, monograph, and asset pages, we often don’t have H1s. We need them. 

Figure out what the H1 should be.
